### PR TITLE
Make the error massage more readable when it is unable to fetch the portal bundle from sdk

### PR DESCRIPTION
**Pull request by jtydhr88**
hey Greg,
I found our error message is hard to read when we got from the error server.
I create the EmptyPortlalBundle to save the more detail when it failed to get from sdk.
Use it instead of "return null"

### CODE REVIEW

File: tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/util/ServerUtil.java [hunk 1]
```diff
@@ -815,7 +787,8 @@ public static PortalBundle getPortalBundle( IProject project ) throws CoreExcept
             }
         }
 
-        return null;
+        return new EmptyPortalBundle(
+            "Unable to get Liferay Bundle by properties 'app.server.type=" + appServerType + "'" );
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/util/ServerUtil.java]: `Unable to determine Liferay Bundle by reading build properties 'app.serve.rtype='`

File: tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/util/ServerUtil.java [hunk 2]
```diff
@@ -794,7 +766,7 @@ public static PortalBundle getPortalBundle( IProject project ) throws CoreExcept
 
         if( sdk == null || !sdk.validate().isOK() )
         {
-            return null;
+            return new EmptyPortalBundle( "Unable to detect plugin sdk or plugin sdk setting are incorrent" );
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/util/ServerUtil.java]: Unable to detect plugins sdk, or plugins-sdks settings are incorrect.

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/SDKClasspathContainerInitializer.java [hunk 3]
```diff
@@ -72,12 +74,11 @@ public void initialize( IPath containerPath, IJavaProject project ) throws CoreE
             throw new CoreException( ProjectCore.createErrorStatus( msg + SDKClasspathContainer.ID ) );
         }
 
-        PortalBundle bundle = ServerUtil.getPortalBundle(project.getProject());
+        PortalBundle bundle = ServerUtil.getPortalBundle( project.getProject() );
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/SDKClasspathContainerInitializer.java]: How does the correct PortalBundle get created if the user fixes the properties?  How does the SDK Classpath COntainer get  re-initialized?   Can you add tests for this?

Comment (jtydhr88) [reply tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/SDKClasspathContainerInitializer.java]: It wounldn't effect on any correct Bundles if the user fixed the wrong.

**Pull request closed without merging**